pages/learn/getting-started.md

```diff
@@ -74,12 +74,13 @@ Alternatively, by clicking on the toggle section of the download button, you als
 <!-- This section is heavily customized based on the device -->
 {{import "getting-started/getDeviceOnDash"}}
 
-## Add release
+## Install the {{ $names.cli.lower }}
+
+Now that we have a device or two connected to a {{ $names.company.lower }} fleet, it's time to deploy code. The recommended way to deploy code is to use {{ $names.cli.lower }}. Follow the instructions below to install {{ $names.cli.lower }} to your machine's operating system.
```

Comment:
```suggestion
Now that we have a device or two connected to a {{ $names.company.lower }} fleet, it's time to deploy some code. We will use the {{ $names.cli.lower }} for this. Follow the instructions below to install it on your computer (development workstation). Choose the tab for your operating system.
```
* As per [style manual](https://github.com/balena-io/docs/blob/v5.30.10/README.md#balena-specific-words), we currently refer to the CLI as "the balena CLI", and `$names.cli.lower` resolves to "balena CLI".
* "time to deploy some code" sounds better than "time to deploy code", no?
* The rest is just my preference. The emphasis on "your computer (development workstation)" is to try and be as clear as possible that they should not install the CLI on their balenaOS device (which some confused users thought they should do in the past), but rather on their computer.

Reply:
In other sentences "your code" is used, should standardize, is some code or your code more accurate